src/index/formatFileStructure/buildGraph.ts

```diff
@@ -1,55 +1,54 @@
 import path from "path";
 import { findEdges } from "./shared/findEdges";
-import { addEdge } from "./buildGraph/addEdge";
+import { addEdgeToGraph } from "./buildGraph/addEdge";
 import { Graph, OldGraph } from "./shared/Graph";
 import { findSharedParent } from "./shared/findSharedParent";
 import { customResolve } from "./shared/customResolve";
+import OS from "os";
 
+/** Builds a graph for a particular set of files. */
 export function buildGraph(files: string[]) {
   const parentFolder = findSharedParent(files);
   const graph: Graph = {};
   const oldGraph: OldGraph = {};
   const totalFiles: string[] = [];
+
   let numForwardSlashes = 0;
   let numBackSlashes = 0;
-  for (let file of files) {
-    if (file === ".git") {
-      continue;
-    }
-    file = path.resolve(file);
-    const start = path.relative(parentFolder, file);
-    if (!(start in oldGraph)) {
-      oldGraph[start] = {
-        oldLocation: file,
-        imports: [],
-      };
-    }
+
+  for (let filePath of files) {
+    if (filePath === ".git") continue;
+
+    filePath = path.resolve(filePath);
+    const start = path.relative(parentFolder, filePath);
     totalFiles.push(start);
-    findEdges(file).forEach(edge => {
-      if (edge[1].includes("/")) {
-        numForwardSlashes++;
-      } else if (edge[1].includes("\\")) {
-        numBackSlashes++;
-      }
 
-      const pathWithExtension = customResolve(edge[1], path.dirname(edge[0]));
+    // check and execute if oldGraph needs to be initialized with a value.
+    const startNotInitialized = !(start in oldGraph);
+    if (startNotInitialized) {
+      oldGraph[start] = { oldLocation: filePath, imports: [] };
+    }
 
-      const end = path.relative(parentFolder, pathWithExtension);
+    // add imports to start
+    for (const edge of findEdges(filePath)) {
+      const [first, last] = edge;
+
+      if (last.includes("/")) numForwardSlashes++;
+      else if (last.includes("\\")) numBackSlashes++;
```

Comment:
I would prefer to use `{` and `}` here since the statement does not exit in anyway.

Reply:
I would prefer to keep it this way because it's one line and super easy to read. 

I think that using parentheses should be used when setting variable/s or multiple lines of code.